x-pack/plugin/esql/qa/testFixtures/src/main/resources/semantic_text.csv-spec

```diff
@@ -0,0 +1,10 @@
+simple
+required_capability: semantic_text_type
+
+FROM semantic_text;
```

Comment:
You'll have to `| SORT` this or else it'll be jumbled in any multi-node tests.

Reply:
good catch. done.